flow/layers/child_scene_layer.cc

```diff
@@ -19,7 +19,9 @@ ChildSceneLayer::~ChildSceneLayer() {
 
 void ChildSceneLayer::Preroll(PrerollContext* context, const SkMatrix& matrix) {
   transform_ = matrix;
-  transform_.postTranslate(offset_.x(), offset_.y());
+  transform_.preTranslate(offset_.x(), offset_.y());
```

Comment:
Yeah, turns out these matrix ops are backwards from what I would have expected.  Apparently "preScale" means M * S, where M is the current matrix and S is the scale matrix.